projects/helper/solana.js

```diff
@@ -137,6 +137,7 @@ async function getTokenAccountBalances(tokenAccounts, { individual = false, chun
           return;
         }
         console.log(data.data.map(i => i.result.value)[i], tokenAccounts[i].toString())
+        return;
```

Comment:
@DonDuala thanks for the PR, I will be reverting this change though,  code should never reach here, if it reaches here and breaks, there is a bug somewhere, dont you agree?

Reply:
We were reaching there. Didnt dig too deep into why it was happening, but since they were only for accounts that did not have any tokens, thought it was fine to just log the error and move on.